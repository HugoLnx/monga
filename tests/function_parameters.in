int[][][] main(int[] nums, char[][] args) {
}
