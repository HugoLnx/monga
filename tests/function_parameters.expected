declarations:
	declaration:
		function:main,259,3
			parameters:
				parameter:nums,259,1
				parameter:args,260,2
			block:
